src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/LongPollingTransport.cs

```diff
@@ -26,13 +26,60 @@ public LongPollingTransport(CancellationToken timeoutToken, PipeReader applicati
             _logger = loggerFactory.CreateLogger<LongPollingTransport>();
         }
 
+        public async Task ProcessFirstRequestAsync(HttpContext context, CancellationToken token)
```

Comment:
When we talked yesterday, the plan was to just remove all the Long Polling logic (set both Client and Server back to their behavior before this PR). Long Polling will be dumb, and the client will just assume `StartAsync` works as before (and thus the server doesn't need to do any flushing). This PR is still relevant for the Server-Sent Events changes, but the Long Polling changes can just be reverted entirely